blocktx/store/sql/primary_blocktx.go

```diff
@@ -25,16 +25,18 @@ func (s *SQL) PrimaryBlocktx(ctx context.Context) (string, error) {
 
 	defer rows.Close()
 
+	var ind int
 	var hostName string
 
 	for rows.Next() {
 		err = rows.Scan(&hostName)
 		if err != nil {
 			return "", err
 		}
-
-		return hostName, nil
+		if ind == 0 {
```

Comment:
ind is by default 0 so the loop will exit on first iteration. Is that intended?

Reply:
Yes, otherwise the linter complained 🤷